Rakefile

```diff
@@ -51,14 +51,8 @@ def linkify_prs(markdown)
   end
 end
 
-def linkify_users(markdown)
-  markdown.gsub(%r!(@\w+)!) do |username|
-    "[#{username}](https://github.com/#{username.delete("@")})"
-  end
-end
-
 def linkify(markdown)
-  linkify_users(linkify_prs(markdown))
+  linkify_prs(markdown)
 end
```

Comment:
So this function is useless (for) now, we can directly call `linkify_prs` in `converted_history`